webapi/src/main/resources/application.conf

```diff
@@ -242,7 +242,7 @@ app {
                 name = "http://www.knora.org/data/beol"
             }
             {
-                path = "_test_data/ontologies/biblio.ttl"
+                path = "_test_data/ontologies/biblio-onto.ttl"
```

Comment:
Do  we need a project for `biblio`? 

`knora-dc` is similar to `biblio`, and it does not have a project, it is just defined like this in `application.conf`: 

```
rdf-data = [
            {
                path = "../knora-ontologies/knora-base.ttl"
                name = "http://www.knora.org/ontology/knora-base"
            }
            {
                path = "../knora-ontologies/knora-dc.ttl"
                name = "http://www.knora.org/ontology/dc"
            } ... ]
```

As far as I understand, resource types and properties from `biblio` can be used among different projects, but there won't be a `biblio` project.

So in my opinion `biblio-onto.ttl` is all we need.